tests/CSharp/CSharp.h

```diff
@@ -1217,3 +1217,14 @@ void* TestIndexedProperties::operator[](size_t n) const
 extern const ComplexArrayElement ArrayOfVariableSize[];
 
 void useStdStringJustAsParameter(std::string s);
+
+typedef void* (typedefedFuncPtr)(Foo *a, Bar b);
+
+class DLL_API HasFuncWithTypedefedFuncPtrAsParam
```

Comment:
We try to keep tests as short as possible so we don't use classes unless we test them. For this bug a free function is enough.